website/client/router.js

```diff
@@ -286,6 +299,8 @@ const store = getStore();
 router.beforeEach(function routerGuard (to, from, next) {
   const isUserLoggedIn = store.state.isUserLoggedIn;
   const routeRequiresLogin = to.meta.requiresLogin !== false;
+  const isUserLoggedServer = store.dispatch('auth:loginCheck');
```

Comment:
this function is async, you should wait for it with await but it also means a full request will be made for every route change on the client which is indesiderable. You should add a route guard function just to the speicific logout route